aptos-move/framework/aptos-framework/sources/atomic_bridge.move

```diff
@@ -1686,4 +1818,65 @@ module aptos_framework::atomic_bridge_counterparty {
 
         complete_bridge_transfer(bridge_transfer_id, b"not the secret");
     }
+
+    #[test(aptos_framework = @aptos_framework, third_party = @0xdead)]
+    /// Test eligible sponsor 
+    fun test_lock_for_eligible_sponsor(aptos_framework: &signer, third_party: &signer) acquires BridgeCounterpartyEvents {
+        initialize_for_test(aptos_framework);
+        initialize(aptos_framework);
+
+        // transfer some Aptos Coin from the third party to the framework
+        aptos_account::create_account(signer::address_of(third_party));
+        atomic_bridge::mint(signer::address_of(third_party), 1_000_000_000);
+        aptos_account::transfer(third_party, signer::address_of(aptos_framework), 1_000_000_000);
+
+        let initiator = valid_eip55();
+        let bridge_transfer_id = valid_bridge_transfer_id();
+        let hash_lock = valid_hash_lock();
+        let recipient = @0xcafe;
+        let amount = atomic_bridge_configuration::minimum_transfer_amount();
+
+        lock_bridge_transfer_assets(aptos_framework,
+            initiator,
+            bridge_transfer_id,
+            hash_lock,
+            recipient,
+            amount);
+
+        let sponsored_amount = atomic_bridge_configuration::sponsorship_amount();
+
+        // check that the recipient has been sponsored
+        assert!(coin::balance<AptosCoin>(recipient) == sponsored_amount, 0);
+
+    }
+
+    #[test(aptos_framework = @aptos_framework, third_party = @0xdead)]
+    /// Test ineligible sponsor
+    fun test_lock_for_ineligible_sponsor(aptos_framework: &signer,  third_party: &signer) acquires BridgeCounterpartyEvents {
```

Comment:
```suggestion
    fun test_lock_for_ineligible_sponsorship(aptos_framework: &signer,  third_party: &signer) acquires BridgeCounterpartyEvents {
```

This isn't a test of whether a specific sponsor is eligible, but whether a transfer is eligible for sponsorship.